pkg/adapter/nimbus-kubearmor/processor/kspbuilder.go

```diff
@@ -221,27 +221,45 @@ func swDeploymentToolsKsp() kubearmorv1.KubeArmorPolicy {
 	}
 }
 
-func disallowCapabilities() kubearmorv1.KubeArmorPolicy {
+func disallowCapabilities(rule v1.Rule) kubearmorv1.KubeArmorPolicy {
+
+	capabilities := []kubearmorv1.MatchCapabilitiesType{
+		{
+			Capability: "sys_admin",
+		},
+		{
+			Capability: "sys_ptrace",
+		},
+		{
+			Capability: "sys_module",
+		},
+		{
+			Capability: "dac_read_search",
+		},
+		{
+			Capability: "dac_override",
+		},
+	}
+
+	exclude_caps := rule.Params["exclude_caps"]
+
+	if len(exclude_caps) > 0 {
+
+		capabilities = []kubearmorv1.MatchCapabilitiesType{}
+
+		for _, cap := range exclude_caps {
+			cap1 := kubearmorv1.MatchCapabilitiesStringType(strings.ToLower(cap))
+			capability := kubearmorv1.MatchCapabilitiesType{
+				Capability: cap1,
+			}
+			capabilities = append(capabilities, capability)
+		}
```

Comment:
Here you're doing opposite of exclusion, we need to exclude the capabilities specified in security intent params from our default capability list.

For e.g.,
```yaml
...
params:
  exclude_caps: ["dac_read_search"]
...
```
then the resultant policy won't contain this capability in its capabilities rules. So the resultant policy will look something like:
```yaml
...
capabilities:
    matchCapabilities:
    - capability: sys_admin
    - capability: sys_ptrace
    - capability: sys_module
    - capability: dac_override
```